redaxo/src/addons/phpmailer/pages/help.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo5
+ *
+ * @var rex_addon $this
+ */
+$file = rex_file::get(rex_path::addon('phpmailer','README.md'));
+$Parsedown = new Parsedown();
+$body =  '<div id="phpmailer">'.$Parsedown->text($file);
```

Comment:
Besser `rex_markdown::factory()->parse()` nutzen, oder gibt es hier spezifische Gründe, direkt Parsedown zu nutzen?

Reply:
Habe es mir in einem anderen AddOn abgeschaut.

Reply:
geändert ;-)